packages/next-codemod/src/transforms/__testfixtures__/next-cookies-spec/edge-api-route.input.js

```diff
@@ -0,0 +1,12 @@
+export default function handler(req) {
+  const cookie = req.cookies.get('token')
+  const { cookies, headers } = req
+  const c = cookies.get('token')
+
+  console.log(req.cookies.get('token'))
+  console.log(cookies.get('token'))
+  cookies.get('token')
```

Comment:
How about a test for `req.cookies.get(foo)` where foo is a variable instead of literal?